src/react-imgix.js

```diff
@@ -116,16 +124,34 @@ export default class ReactImgix extends Component {
     };
   };
 
+  imgixParams = () => {
+    const params = {
+      ...defaultImgixParams,
+      ...this.props.imgixParams
+    };
+
+    const { faces, entropy } = params;
```

Comment:
I feel a little bit uneasy about encouraging users to put "non-standard" parameters into the `imgixParams` object. I have no problem putting "helper" options into the parent `this.props` object, but if we're going to explicitly call this a map of "imgix params" then I feel like we should require users to pass in `crop: 'faces'` instead of `faces: true`.

Reply:
I had actually forgotten that `faces` wasn't a top level imgix parameter. I agree with you, I feel uneasy about it as well. I'll remove `faces` in favour of using `crop: 'faces'` directly.